docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -179,6 +179,13 @@ as root:
   kubeadm join --token <token> <master-ip>:<master-port> --discovery-token-ca-cert-hash sha256:<hash>
 ```
 
+To let kubectl work in the following steps, you need to run these commands as a regular user (which is part of the output above):
```

Comment:
suggest: To make kubectl work for your non-root user, you might want to run these commands (which also is a part of the `kubeadm init` output):